ui/package.json

```diff
@@ -128,7 +128,7 @@
     "junit-viewer": "^4.11.1",
     "mocha": "^5.2.0",
     "mocha-junit-reporter": "^1.18.0",
-    "parcel": "^1.11.0",
+    "parcel": "^1.12.3",
```

Comment:
we'll have to fix HMR, but that seems OK. I spiked on a solution a few weeks back.